options/posix/generic/sys-resource-stubs.cpp

```diff
@@ -42,6 +42,8 @@ int getrlimit(int resource, struct rlimit *limit) {
 	return 0;
 }
 
+[[gnu::alias("getrlimit")]] int getrlimit64(int resource, struct rlimit *limit);
```

Comment:
Should it be behind an ifdef for 32 bit platforms?

Reply:
No, this is intended to be present on 32-bit platforms as well.